worker/buildbot_worker/scripts/upgrade_worker.py

```diff
@@ -20,10 +20,10 @@
 from buildbot_worker.scripts import base
 
 
-def upgradeSlave(config):
+def upgradeWorker(config):
     basedir = os.path.expanduser(config['basedir'])
 
-    if not base.isBuildslaveDir(basedir):
+    if not base.isWorkerDir(basedir):
         return 1
 
     buildbot_tac = open(os.path.join(basedir, "buildbot.tac")).read()
```

Comment:
IMO, `upgrade-worker` *may* not be completely dropped from sources (since perhaps at some point we will need such `upgrade-worker`), but for current version this command at least should be disabled, removed from help message, removed from docs, etc.

But I would prefer to drop it completely — less code, less to maintain.